packages/smart-contracts/src/lib/artifacts/ChainlinkConversionPath/0.1.0.json

```diff
@@ -231,7 +231,7 @@
       "outputs": [
         {
           "internalType": "uint256",
-          "name": "result",
+          "name": "rate",
```

Comment:
won't modifying this potentially cause issue with the deployed contract?

Reply:
Nope, and it was deployed with "rate" on mainnet